torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -138,66 +162,89 @@ bool frozenMkldnnCompatibleConvNode(Node* n) {
   }
 
   auto weight = constant_as<Tensor>(n->namedInput("weight")).value();
-  if (!weight.device().is_cpu()) {
-    return false;
+  return supportedMKLDNNWeight(weight);
+}
+
+void replaceInputWithMKLDNNTensor(
+    Node* n,
+    std::string name,
+    const at::Tensor& mkldnn_tensor) {
+  Value* input = n->namedInput(name);
+  auto mkldnn_tensor_value =
+      createConstantMKLDNNTensorOp(n->owningGraph(), mkldnn_tensor)
+          ->insertBefore(n)
+          ->output();
+  mkldnn_tensor_value->setDebugName(input->debugName() + "_mkldnn");
+  n->replaceInputWith(input, mkldnn_tensor_value);
+}
+
+void replaceInputWithMKLDNNTensor(Node* n, std::string name) {
+  Value* input = n->namedInput(name);
+  auto mkldnn_tensor = constant_as<Tensor>(input)->to_mkldnn();
+  replaceInputWithMKLDNNTensor(n, name, mkldnn_tensor);
+}
+
+void convertFrozenConvToMKLDNN(Node* conv) {
```

Comment:
did anything change in here from the previous PR?

Reply:
No, just refactoring..